html/pages/device/ports/fdb.inc.php

```diff
@@ -7,7 +7,9 @@
             <th data-column-id="mac_address">MAC address</th>
             <th data-column-id="ipv4_address">IPv4 Address</th>
             <th data-column-id="interface">Port</th>
+            <th data-column-id="description">Description</th>
             <th data-column-id="vlan">Vlan</th>
+            <th data-column-id="dnsname">DNS Name</th>
```

Comment:
For me this is fairly useless so just has N/A throughout. I imagine it's useful for others but not all.

Reply:
Hidden by default now.